modules/measure/src/main/java/com/opengamma/strata/measure/swaption/DefaultSwaptionMarketDataLookup.java

```diff
@@ -192,11 +222,7 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    StringBuilder buf = new StringBuilder(64);
-    buf.append("DefaultSwaptionMarketDataLookup{");
-    buf.append("volatilityIds").append('=').append(JodaBeanUtils.toString(volatilityIds));
-    buf.append('}');
-    return buf.toString();
+    return "DefaultSwaptionMarketDataLookup{" + "volatilityIds" + '=' + JodaBeanUtils.toString(volatilityIds) + '}';
```

Comment:
Intellij warned me about replacing string builder with string. Wasn't too sure about it though, but if there is no performance issues it odes look more readable.